models/intel/smartlab-action-recognition-0001/accuracy-check.yml

```diff
@@ -0,0 +1,34 @@
+evaluations:
+  - name: smartlab-action-recognition-0001
```

Comment:
please remove this file, I will provide my own config in separated request

Reply:
removed